salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -55,10 +59,23 @@ git pull || error_and_exit "Can not pull hugo-course-publisher"
 yarn install --pure-lockfile \
     || error_and_exit "Can not install hugo-course-publisher"
 
-log_message "Pulling ocw-to-hugo and source data"
+log_message "Clearing site/content directory"
+rm -rf site/content/*
+if [ $? -ne 0 ]; then
+    error_and_exit "Could not clear site/content directory"
+fi
+
+log_message "Clearing node_modules directory"
```

Comment:
This is great, but I think we need to run `yarn install` after clearing out `node_modules` to ensure that everything is installed before the import / build.

Reply:
OK. I got those things out of order. Will re-order them.

Reply:
Changed in commit 00bfde3d